utils/generatePairs.ts

```diff
@@ -50,4 +50,31 @@ function shuffle(array: string[]): string[] {
   return array;
 }
 
-export default generatePairs;
\ No newline at end of file
+export const shiftByOne = async (): Promise<string[][]>  => {
+  const fs = require('fs');
+  let lastPairings = require('./data/lastPairings.json');
+
+  // get first person in each pairing
+  const firstPersonList = lastPairings.map((pairs: string[]) => pairs[0])
+
+  // put first person at end of list (shifting everyone up one)
+  const firstElem = firstPersonList.shift()
+  if (firstElem) {
+    firstPersonList.push(firstElem)
+  }
+
+  // update pairings with new first person
+  lastPairings = lastPairings.map((pairs: string[], ind: number) => [firstPersonList[ind], pairs[1]])
```

Comment:
does this work for odd number of people? specifically, in the case where a pairing could have 3 members

Reply:
O lmao that's so funny the 2nd and 3rd person will always be together :)

Reply:
I propose we just don't let there be an odd number of people